src/app/home/home.component.scss

```
@@ -0,0 +1,67 @@
+$color-dark: #212529;
+$app-background-color: #d3d3d3;
+$color-white: #ffffff;
```

import and reuse variables 
```suggestion
@import ....
```